### ISSUE TITLE
[datadog_gql_link]Multipart request doesn't work

### ISSUE THREAD

**Issue by atsumo** [DataDog/dd-sdk-flutter#618]
### Describe the bug

When I try to execute a mutation with a multipart request, an error occurs during the jsonEncode process of datadog_gql_link, causing the entire operation to fail.

### Reproduction steps

```
final graphQlUrl = "https://example.com/graphql";

final link = Link.from([
  DatadogGqlLink(DatadogSdk.instance, Uri.parse(graphQlUrl)),
  HttpLink(graphQlUrl),
]);
```

when request MultipartFile parsed mutation

### Affected SDK versions

1.0.0 ~

### Did you confirm if the latest SDK version fixes the bug?

Yes

**Comment by fuzzybinary** [comment]
Hi @atsumo 

Do you get a stack trace that you can share?

**Comment by atsumo** [comment]
@fuzzybinary 
I apologize for not sharing the error message.
This is error message

```
 flutter: Converting object to an encodable object failed: Instance of 'MultipartFile'
 flutter: #0      _JsonStringifier.writeObject (dart:convert/json.dart:793:7)
           #1      _JsonStringifier.writeMap (dart:convert/json.dart:874:7)
           #2      _JsonStringifier.writeJsonValue (dart:convert/json.dart:829:21)
           #3      _JsonStringifier.writeObject (dart:convert/json.dart:784:9)
           #4      _JsonStringifier.writeList (dart:convert/json.dart:841:7)
           #5      _JsonStringifier.writeJsonValue (dart:convert/json.dart:823:7)
           #6      _JsonStringifier.writeObject (dart:convert/json.dart:784:9)
           #7      _JsonStringifier.writeMap (dart:convert/json.dart:874:7)
           #8      _JsonStringifier.writeJsonValue (dart:convert/json.dart:829:21)
           #9      _JsonStringifier.writeObject (dart:convert/json.dart:784:9)
           #10     _JsonStringStringifier.printOn (dart:convert/json.dart:982:17)
           #11     _JsonStringStringifier.stringify (dart:convert/json.dart:967:5)
           #12     JsonEncoder.convert (dart:convert/json.dart:345:30)
           #13     JsonCodec.encode (dart:convert/json.dart:231:45)
           #14     jsonEncode (dart:convert/json.dart:114:10)
           #15   <…>
```

```
"Unhandled Client-Side Exception: Converting object to an encodable object failed: Instance of 'MultipartFile'"
```

**Comment by fuzzybinary** [comment]
Okay, I'll try to take a look this week.

**Comment by fuzzybinary** [comment]
Sorry can you also provide me with an example mutation that you're running?

**Comment by fuzzybinary** [comment]
This should now be fixed in v1.1.1

**Comment by atsumo** [comment]
@fuzzybinary 
Thank you I will try it!
